bigchaindb/config_utils.py

```diff
@@ -287,3 +287,16 @@ def load_consensus_plugin(name=None):
                         'consensus.BaseConsensusRules`'.format(type(plugin)))
 
     return plugin
+
+
+def load_block_publisher_plugins(names):
+    plugins = []
+
+    if names is None:
```

Comment:
We need to decide what the no plugins default should be (`[]` or `None`).
https://github.com/bigchaindb/bigchaindb/pull/1707/files#diff-38ba73c9ead7452abd625b636f098e99R64 is using `[]` as the default if the config is not set.